doc/src/manual/interacting-with-julia.md

```diff
@@ -61,44 +61,60 @@ at detecting when a paste occurs.
 ### Help mode
 
 When the cursor is at the beginning of the line, the prompt can be changed to a help mode by typing
-`?`.  Julia will attempt to print help or documentation for anything entered in help mode:
+`?`. Julia will attempt to print help or documentation for anything entered in help mode:
 
 ```julia
-julia> ? # upon typing ?, the prompt changes (in place) to: help>
+julia> ? # upon typing ?, the prompt changes (in place) to: help?>
 
-help> string
-Base.string(xs...)
+help?> string
+search: string String stringmime Cstring Cwstring RevString readstring randstring bytestring SubString
 
-   Create a string from any values using the "print" function.
+  string(xs...)
+
+  Create a string from any values using the print function.
 ```
 
 In addition to function names, complete function calls may be entered to see which method is called
-for the given argument(s).  Macros, types and variables can also be queried:
+for the given argument(s). Macros, types and variables can also be queried:
 
 ```
-help> string(1)
```

Comment:
the output probably hadn't been touched since several iterations ago of the doc system?